quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/LoggingPropertyMappers.java

```diff
@@ -29,15 +31,18 @@ public static PropertyMapper<?>[] getMappers() {
                         .paramLabel("<handler>")
                         .build(),
                 fromOption(LoggingOptions.LOG_CONSOLE_OUTPUT)
+                        .isEnabled(LoggingPropertyMappers::isConsoleEnabled, "Console log handler is activated")
```

Comment:
It would be good to collect the whenEnabled constants to single variables.

Reply:
Done here:
https://github.com/keycloak/keycloak/pull/25333/files#diff-78e1ea4992f437dbda8637131776d9f1e448a2cf4f6ba2dd986e07ec0d1eb298R26-R28